postmaster/programmer/FirmwareReceptorResetTarget.cpp

```diff
@@ -10,19 +10,38 @@ namespace application
 
     void FirmwareReceptorResetTarget::ReceptionStarted()
     {
-        {
-            hal::OutputPin activateReset(reset, false);
-            activateBoot0.Emplace(boot0, true);
-
-            delegate.Emplace(delegateCreator);
-        }
-
-        (*delegate)->ReceptionStarted();
+        starting = true;
+        activateReset.Emplace(reset, false);
+        activateBoot0.Emplace(boot0, true);
+
+        timer.Start(std::chrono::milliseconds(1), [this]()
+            {
+                activateReset->Set(true);
+
+                timer.Start(std::chrono::milliseconds(1), [this]()
+                    {
+                        activateReset = infra::none;
```

Comment:
Here you should actively drive low
```suggestion
                        activateReset->Set(false);
```

Reply:
Yes, but then potentially nothing drives the Pin anymore which and it might take several 10s of `ms` until the Reset is at a level the MCU will start up. At least that's the case with `NUCLEO-H563ZI` board

Reply:
You're right, my mistake